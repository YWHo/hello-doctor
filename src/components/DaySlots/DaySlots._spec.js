import React from 'react';
import dayjs from 'dayjs';
import { shallow } from 'enzyme';
import { DaySlots } from './DaySlots';

describe('DaySlots', () => {
  it('render correctly', () => {
    const wrapper = shallow(
      <DaySlots
        selectedDate={dayjs('2019-06-19T00:37:25+0000')}
        today={dayjs('2019-06-16T10:17:13+0000')}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
