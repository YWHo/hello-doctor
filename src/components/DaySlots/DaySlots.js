import React from 'react';
import { connect } from 'react-redux';
import styled from 'styled-components';
import dayjs from 'dayjs';
import * as actions from '../../state/actions';
import * as selectors from '../../state/selectors';
import * as dateHelper from '../../shared/dateHelper';

const ContainerOuter = styled.div`
  display: block;
  height: 78px;
  padding: 12px 2px;
  overflow: hidden;
  white-space: nowrap;
`;

const ContainerInner = styled.div`
  display: block;
  height: 84px;
  overflow-x: auto;
  overflow-y: hidden;
  padding: 0 2px;
  white-space: nowrap;
`;

const ButtonCircle = styled.button`
  background-color: ${props => (props.selected ? '#177d91' : 'transparent')};
  border: none;
  border-radius: 50%;
  color: #fff;
  height: 66px;
  width: 66px;
  margin-right: 8px;

  :focus {
    outline: none;
  }

  :active {
    background-color: #1c98b0;
    outline: none;
  }
`;

const DateNum = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  height: 27px;
  font-family: 'Roboto', sans-serif;
  font-size: 24px;
`;

const WeekDayName = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  height: 16px;
  font-size: 14px;
  font-weight: 300;
`;

export function DaySlots(props) {
  const { selectedDate = dayjs() } = props;

  const days = dateHelper.getDatesInThreeMonthsFrom();
  const dateList = days.map((day, idx) => {
    const tmpDate = dayjs(day);
    const selected = day === dayjs(selectedDate).format('YYYY-MM-DD');
    return (
      <ButtonCircle
        key={`dateCi_${idx}`}
        selected={selected}
        onClick={() => onDayClicked(props, day)}
      >
        <DateNum key={`dateNu_${idx}`}>{tmpDate.format('D')}</DateNum>
        <WeekDayName key={`weekDa_${idx}`}>
          {tmpDate.format('ddd').toLowerCase()}
        </WeekDayName>
      </ButtonCircle>
    );
  });

  return (
    <ContainerOuter>
      <ContainerInner>{dateList}</ContainerInner>
    </ContainerOuter>
  );
}

function onDayClicked(props, day) {
  const { dSaveSelectedDate } = props;
  dSaveSelectedDate(dayjs(day));
}

const mapStateToProps = state => {
  return {
    selectedDate: selectors.getSelectedDate(state)
  };
};

function mapDispatchToProps(dispatch) {
  return {
    dSaveSelectedDate: value => dispatch(actions.saveSelectedDate(value))
  };
}

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(DaySlots);
