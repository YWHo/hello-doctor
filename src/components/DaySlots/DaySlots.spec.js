import React from 'react';
import dayjs from 'dayjs';
import { shallow } from 'enzyme';
import { DaySlots } from './DaySlots';

describe('DaySlots', () => {
  it('render correctly', () => {
    const wrapper = shallow(
      <DaySlots
        selectedDate={dayjs('2019-06-19')}
        today={dayjs('2019-06-16')}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
