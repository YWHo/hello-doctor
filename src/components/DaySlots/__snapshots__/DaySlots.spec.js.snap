// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DaySlots render correctly 1`] = `
<styled.div>
  <styled.div>
    <styled.button
      key="dateCi_0"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_0"
      >
        16
      </styled.div>
      <styled.div
        key="weekDa_0"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_1"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_1"
      >
        17
      </styled.div>
      <styled.div
        key="weekDa_1"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_2"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_2"
      >
        18
      </styled.div>
      <styled.div
        key="weekDa_2"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_3"
      onClick={[Function]}
      selected={true}
    >
      <styled.div
        key="dateNu_3"
      >
        19
      </styled.div>
      <styled.div
        key="weekDa_3"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_4"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_4"
      >
        20
      </styled.div>
      <styled.div
        key="weekDa_4"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_5"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_5"
      >
        21
      </styled.div>
      <styled.div
        key="weekDa_5"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_6"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_6"
      >
        22
      </styled.div>
      <styled.div
        key="weekDa_6"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_7"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_7"
      >
        23
      </styled.div>
      <styled.div
        key="weekDa_7"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_8"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_8"
      >
        24
      </styled.div>
      <styled.div
        key="weekDa_8"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_9"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_9"
      >
        25
      </styled.div>
      <styled.div
        key="weekDa_9"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_10"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_10"
      >
        26
      </styled.div>
      <styled.div
        key="weekDa_10"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_11"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_11"
      >
        27
      </styled.div>
      <styled.div
        key="weekDa_11"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_12"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_12"
      >
        28
      </styled.div>
      <styled.div
        key="weekDa_12"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_13"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_13"
      >
        29
      </styled.div>
      <styled.div
        key="weekDa_13"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_14"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_14"
      >
        30
      </styled.div>
      <styled.div
        key="weekDa_14"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_15"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_15"
      >
        1
      </styled.div>
      <styled.div
        key="weekDa_15"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_16"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_16"
      >
        2
      </styled.div>
      <styled.div
        key="weekDa_16"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_17"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_17"
      >
        3
      </styled.div>
      <styled.div
        key="weekDa_17"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_18"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_18"
      >
        4
      </styled.div>
      <styled.div
        key="weekDa_18"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_19"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_19"
      >
        5
      </styled.div>
      <styled.div
        key="weekDa_19"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_20"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_20"
      >
        6
      </styled.div>
      <styled.div
        key="weekDa_20"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_21"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_21"
      >
        7
      </styled.div>
      <styled.div
        key="weekDa_21"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_22"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_22"
      >
        8
      </styled.div>
      <styled.div
        key="weekDa_22"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_23"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_23"
      >
        9
      </styled.div>
      <styled.div
        key="weekDa_23"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_24"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_24"
      >
        10
      </styled.div>
      <styled.div
        key="weekDa_24"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_25"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_25"
      >
        11
      </styled.div>
      <styled.div
        key="weekDa_25"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_26"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_26"
      >
        12
      </styled.div>
      <styled.div
        key="weekDa_26"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_27"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_27"
      >
        13
      </styled.div>
      <styled.div
        key="weekDa_27"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_28"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_28"
      >
        14
      </styled.div>
      <styled.div
        key="weekDa_28"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_29"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_29"
      >
        15
      </styled.div>
      <styled.div
        key="weekDa_29"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_30"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_30"
      >
        16
      </styled.div>
      <styled.div
        key="weekDa_30"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_31"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_31"
      >
        17
      </styled.div>
      <styled.div
        key="weekDa_31"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_32"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_32"
      >
        18
      </styled.div>
      <styled.div
        key="weekDa_32"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_33"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_33"
      >
        19
      </styled.div>
      <styled.div
        key="weekDa_33"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_34"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_34"
      >
        20
      </styled.div>
      <styled.div
        key="weekDa_34"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_35"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_35"
      >
        21
      </styled.div>
      <styled.div
        key="weekDa_35"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_36"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_36"
      >
        22
      </styled.div>
      <styled.div
        key="weekDa_36"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_37"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_37"
      >
        23
      </styled.div>
      <styled.div
        key="weekDa_37"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_38"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_38"
      >
        24
      </styled.div>
      <styled.div
        key="weekDa_38"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_39"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_39"
      >
        25
      </styled.div>
      <styled.div
        key="weekDa_39"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_40"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_40"
      >
        26
      </styled.div>
      <styled.div
        key="weekDa_40"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_41"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_41"
      >
        27
      </styled.div>
      <styled.div
        key="weekDa_41"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_42"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_42"
      >
        28
      </styled.div>
      <styled.div
        key="weekDa_42"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_43"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_43"
      >
        29
      </styled.div>
      <styled.div
        key="weekDa_43"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_44"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_44"
      >
        30
      </styled.div>
      <styled.div
        key="weekDa_44"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_45"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_45"
      >
        31
      </styled.div>
      <styled.div
        key="weekDa_45"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_46"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_46"
      >
        1
      </styled.div>
      <styled.div
        key="weekDa_46"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_47"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_47"
      >
        2
      </styled.div>
      <styled.div
        key="weekDa_47"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_48"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_48"
      >
        3
      </styled.div>
      <styled.div
        key="weekDa_48"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_49"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_49"
      >
        4
      </styled.div>
      <styled.div
        key="weekDa_49"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_50"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_50"
      >
        5
      </styled.div>
      <styled.div
        key="weekDa_50"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_51"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_51"
      >
        6
      </styled.div>
      <styled.div
        key="weekDa_51"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_52"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_52"
      >
        7
      </styled.div>
      <styled.div
        key="weekDa_52"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_53"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_53"
      >
        8
      </styled.div>
      <styled.div
        key="weekDa_53"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_54"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_54"
      >
        9
      </styled.div>
      <styled.div
        key="weekDa_54"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_55"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_55"
      >
        10
      </styled.div>
      <styled.div
        key="weekDa_55"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_56"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_56"
      >
        11
      </styled.div>
      <styled.div
        key="weekDa_56"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_57"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_57"
      >
        12
      </styled.div>
      <styled.div
        key="weekDa_57"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_58"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_58"
      >
        13
      </styled.div>
      <styled.div
        key="weekDa_58"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_59"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_59"
      >
        14
      </styled.div>
      <styled.div
        key="weekDa_59"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_60"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_60"
      >
        15
      </styled.div>
      <styled.div
        key="weekDa_60"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_61"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_61"
      >
        16
      </styled.div>
      <styled.div
        key="weekDa_61"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_62"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_62"
      >
        17
      </styled.div>
      <styled.div
        key="weekDa_62"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_63"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_63"
      >
        18
      </styled.div>
      <styled.div
        key="weekDa_63"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_64"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_64"
      >
        19
      </styled.div>
      <styled.div
        key="weekDa_64"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_65"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_65"
      >
        20
      </styled.div>
      <styled.div
        key="weekDa_65"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_66"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_66"
      >
        21
      </styled.div>
      <styled.div
        key="weekDa_66"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_67"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_67"
      >
        22
      </styled.div>
      <styled.div
        key="weekDa_67"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_68"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_68"
      >
        23
      </styled.div>
      <styled.div
        key="weekDa_68"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_69"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_69"
      >
        24
      </styled.div>
      <styled.div
        key="weekDa_69"
      >
        sat
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_70"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_70"
      >
        25
      </styled.div>
      <styled.div
        key="weekDa_70"
      >
        sun
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_71"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_71"
      >
        26
      </styled.div>
      <styled.div
        key="weekDa_71"
      >
        mon
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_72"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_72"
      >
        27
      </styled.div>
      <styled.div
        key="weekDa_72"
      >
        tue
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_73"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_73"
      >
        28
      </styled.div>
      <styled.div
        key="weekDa_73"
      >
        wed
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_74"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_74"
      >
        29
      </styled.div>
      <styled.div
        key="weekDa_74"
      >
        thu
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_75"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_75"
      >
        30
      </styled.div>
      <styled.div
        key="weekDa_75"
      >
        fri
      </styled.div>
    </styled.button>
    <styled.button
      key="dateCi_76"
      onClick={[Function]}
      selected={false}
    >
      <styled.div
        key="dateNu_76"
      >
        31
      </styled.div>
      <styled.div
        key="weekDa_76"
      >
        sat
      </styled.div>
    </styled.button>
  </styled.div>
</styled.div>
`;
